src/assets/scss/partials/global/_header.scss

```diff
@@ -0,0 +1,194 @@
+
+.rh-design {
+
+    li.story a{
+        padding-left: 4px;
+    }
+
+    li.story a.active{
+        background: $gray-100;
```

Comment:
As a good habit, whenever defining just a background color go ahead and use `background-color` instead of `background` because it poses a risk when you're defining other background properties with `background` and then use `background` again just to change the background color. Here's a good explanation:

"If you’ve previously applied various `background` declarations on an element, e.g. background-color or background-image, and then later on add a `background: red` declaration to the same element, all your previous declarations will be reset, and now your element’s background is red, and your background image is gone."

So unless you're defining multiple properties, I'd just stick with using the exact property that you're trying to change. In this case, `background-color`. I do see that you've done this already in some places, but it's inconsistent. Try a find and replace, to ensure I haven't missed any instances (I'm seeing 4 total).

Reply:
Updated